transcom/mymove
# B 21532 main

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21532)

## Summary
When an office user is requesting an account for MilMove, they need the option to select HQ User on their registration form.

### How to test Frontend

1. Access [milmove](http://officelocal:3000/sign-in)
2. select request access in the top left corner
3. Create user and Verify the "headquarters" checkbox is present and working.

### How to test Backend
1. Once user has been created open the office users table in Dbeaver
2. Filter the user_roles table with the office user ID of the newly created user. 
3. Verify the role is now "HeadQuarters"

### 508 Screenshots
[B-21532-508Screenshots.pdf](https://github.com/user-attachments/files/17563888/B-21532-508Screenshots.pdf)


## [21532 int Add HW Role to Request Account Screen](https://github.com/transcom/mymove/pull/14056)
